lisp/ess-tracebug.el

```diff
@@ -122,8 +122,6 @@ Indicates that ess-tracebug-mode is turned on.")
 
 (defvar ess-watch-mode-map
   (let ((map (make-sparse-keymap)))
-    (set-keymap-parent map special-mode-map)
-    (define-key map "?" 'ess-watch-help)
```

Comment:
Why ? was removed?

Reply:
`?` is bound by default in modes derived from special-mode to `describe-mode` which does basically the same thing.